src/tiktok_uploader/upload.py

```diff
@@ -180,7 +180,7 @@ def complete_upload_form(driver, path: str, description: str, schedule: datetime
     _go_to_upload(driver)
     #  _remove_cookies_window(driver)
     _set_video(driver, path=path, **kwargs)
-    _remove_split_window(driver)
+    #_remove_split_window(driver)
```

Comment:
I don't think it is a good idea to put this line in comment, in fact if it was added I am assuming that it was necessary or very useful for some people.

I suggest to instead to make this line skipable for people who desires (like me because I am not using it)
```suggestion
   if (!skip_remove_split_window_fct)
    _remove_split_window(driver)
```